frontend/src/components/Comment/CommentList.js

```diff
@@ -1,28 +1,36 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useLayoutEffect } from 'react'
 import { getCommentByPicture } from './CommentAPI'
 import { Comment } from './Comment'
 import './SCSS/CommentList.scss'
+import Axios from 'axios'
 
 const CommentList = (props) => {
 
     const [commentList, setCommentList] = useState([])
+    const [refreshCommentList, setRefreshCommentList] = useState(false);
+    const [loading, setLoading] = useState(true);
 
     useEffect(() => {
         loadComments();
-    }, [props.postId, props.refreshComment]);
+    }, [props.postId, props.refreshComment, refreshCommentList]);
 
     const loadComments = () => {
-        getCommentByPicture(props.postId).then((response) => {
```

Comment:
This should be inside useEffect to prevent a warning

Reply:
I've move this in #76 so I won't be making the change in this branch